apps/issuer/api.py

```diff
@@ -1442,3 +1579,276 @@ def delete(self, request, **kwargs):
             return Response(
                 {"detail": "Staff request not found"}, status=status.HTTP_404_NOT_FOUND
             )
+
+
+class NetworkInvitation(BaseEntityDetailView):
+    model = NetworkInvite
+    v1_serializer_class = NetworkInviteSerializer
+    permission_classes = [
```

Comment:
Aren't we missing at least `IsEditor` here?
Creating, deleting and updating (accepting) invites should be restricted to owners as well.